### PR TITLE
➕ Add Arabic Language Translation

### LINKED ISSUE #752: 💡 Arabic language support
Arabic language support, I can do the translation as a native Arabic speaker. Not weird Google Translate one 😂

### PR DESCRIPTION
**Pull request by YazeedAlKhalaf**
This PR adds Arabic language translation and also adds the language to the `LanguageSelector.ts` 🚀

fixes #752

### CODE REVIEW

**Comment by YazeedAlKhalaf** [comment]
If another guy who knows Arabic can proofread it, it would be great 🙏🚀

**Review by willdoescode — APPROVED** [review]

File: kofta/public/locales/ar/translation.json [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+		"followList": {},
+		"home": { "createRoom": "ابدأ غرفة" },
+		"inviteList": {
+			"roomGone": "انتهت الغرفة، عد إلى الوراء",
+			"shareRoomLink": "شارك رابط الغرفة",
+			"inviteFollowers": "يمكنك أن تدعو متابعيك المتصلين:",
+			"whenFollowersOnline": "سيظهر متابعوك هنا عندما يكونوا متصلين."
+		},
+		"login": {
+			"headerText": "اصطخاب المحادثات الصوتية إلى القمر 🚀",
```
Comment (muhammadsammy) [inline kofta/public/locales/ar/translation.json:48]: typo in "اصطخاب"
correction: "اصطحاب"

File: kofta/public/locales/ar/translation.json [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+			"featureText_1": "المظهر الداكن",
+			"featureText_2": "تسجيل مفتوج للجميع",
```
Comment (muhammadsammy) [inline kofta/public/locales/ar/translation.json:50]: مفتوح

File: kofta/public/locales/ar/translation.json [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+			"featureText_3": "دعم منصات متعددة",
+			"featureText_4": "مفتوح المصدر",
+			"featureText_5": "محادثات نصية",
+			"featureText_6": "مدعوم من Doge",
+			"loginGithub": "التسجيل باستخدام GitHub",
+			"loginTwitter": "التسجيل باستخدام Twitter",
+			"createTestUser": "إنشاء مستخدم اختبار"
```
Comment (muhammadsammy) [inline kofta/public/locales/ar/translation.json:57]: تجريبي بدلا من اختبار

File: kofta/public/locales/ar/translation.json [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+			"refresh": "تحديث قائمة الميكروفونات",
+			"volume": "ارتفاع الصوت:"
+		}
+	},
+	"components": {
+		"_comment": "Component UI Internationalization Strings",
+		"avatar": {},
+		"backBar": {},
+		"blockedFromRoomUsers": {
+			"header": "المستخدمون المظورون",
```
Comment (muhammadsammy) [inline kofta/public/locales/ar/translation.json:97]: typo
المحظورون

File: kofta/public/locales/ar/translation.json [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+		"addToCalendar": { "add": "أضف إلى الرزنامة" },
+		"wsKilled": {
+			"description": "تم قتل الـWebsocket بواسطة الخادم. يحدث هذا عادةً عند فتح موقع الويب في علامة تبويب أخرى.",
+			"reconnect": "أعد الاتصال"
+		},
+		"modals": {
+			"createRoomModal": {
+				"public": "عام",
+				"private": "خاص",
+				"roomName": "السم الغرفة",
```
Comment (muhammadsammy) [inline kofta/public/locales/ar/translation.json:142]: typo
اسم

**Review by muhammadsammy — CHANGES_REQUESTED** [review]
Otherwise, LGTM. Good work!

**Comment by nadirabbas** [comment]
could you pull from `staging` and merge into `staging`?

**Comment by YazeedAlKhalaf** [comment]
> could you pull from `staging` and merge into `staging`?

What do you mean?

Both are at the same level, even

**Comment by nadirabbas** [comment]
this PR actually pulled from `new-design`. and `staging` actually gets deployed 😅

**Comment by YazeedAlKhalaf** [comment]
Ah okay 😎😅, so I just need to make a new PR merging into staging not new design, right? 🤔

**Comment by nadirabbas** [comment]
no, actually i could have changed the base for that. but you need to use `staging` as the source branch. currently this branch was checked out from the `new-design` branch.

**Comment by YazeedAlKhalaf** [comment]
Okay. No problem, will do it now 🚀, thanks 😊

**Pull request closed without merging**